### ISSUE TITLE
[BUG]: sft training 13B model outputs 75go model

### ISSUE THREAD

**Issue by allaccs** [hpcaitech/ColossalAI#4308]
### 🐛 Describe the bug

I trained stage 1 on llama 13B, but the output model is divided in 3*26GO shards. Now I want to fine-tune a second epoch on the  model but it is impossible to load even on 8*A100 80go...

`torchrun --standalone --nproc_per_node=8 train_sft.py \
    --pretrain "/workspace/ColossalAI/applications/Chat/weights/output/" \
    --model 'llama' \
    --strategy colossalai_zero2 \
    --log_interval 20 \
    --save_path  /workspace/ColossalAI/applications/Chat/output/ \
    --dataset /workspace/ColossalAI/applications/Chat/tt_filtered.json \
    --batch_size 2 \
    --accumulation_steps 4 \
    --lr 2e-5 \
    --max_epochs 1 `

The 75GO size doesn't seem right, even the metadata shows "total_size": 26031733760
On the other hand, model works for inference.

I trained on 8*A100 80go. I'm now stuck with my 75GO model.. What can I do, is there a way to reduce it back to 26go? Why would the output be so big?

**Comment by allaccs** [comment]
Removing optimizer and gradient information still gives me a 50go model. Looking at other fined-tuned models they all are around 26GO for the 13B model.
